packages/kbn-build/src/utils/Package.js

```diff
@@ -0,0 +1,130 @@
+import path from 'path';
+import semver from 'semver';
```

Comment:
__nit:__ `semver` is not used, and seems it's the only place that requires this package, so we can probably remove it from `package.json` as well.